app/services/tagging/tagger.rb

```diff
@@ -0,0 +1,14 @@
+module Tagging
+  class Tagger
+    def self.add_tags(content_id, taxon_ids)
+      link_content = Services.publishing_api.get_links(content_id)
+      version = link_content['version']
+      taxons = (link_content.dig('links', 'taxons') || []) + taxon_ids
```

Comment:
Maybe add a `.uniq` in here somewhere? That I think would make this idempotent... although saying that, I guess the Publishing API might do this already.